docs/AboutUs.md

```diff
@@ -5,57 +5,35 @@
 
 # About Us
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
-
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+We are a team taking the Software Engineering (CS2310T) course offered by [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
```

Comment:
Might want to fix the typo CS2310T to CS2103T

Reply:
thanks, changed.